src/TConsole.cpp

```diff
@@ -1681,6 +1683,10 @@ void TConsole::print(const QString& msg)
     buffer.append(msg, 0, msg.size(), mFormatCurrent.foreground(), mFormatCurrent.background(), mFormatCurrent.allDisplayAttributes());
     mUpperPane->showNewLines();
     mLowerPane->showNewLines();
+
+    if (Q_UNLIKELY(mirrorToStdout)) {
+        qDebug().nospace().noquote() << QStringLiteral("%1| %2").arg(mConsoleName, msg);
```

Comment:
:question: 

Technical point - are we ever going to be testing with more than one profile active? If so then we need to provide the profile name as well (I suggest BEFORE the other two arguments)...

\<aside>Could this feature also be of use for those with Visual Impairment prior to full screen-reader interfacing/support being completed?\</aside>

Reply:
Could be, but I'll tackle that later. Right now adding the profile name would just add spam...

Not useful for visual support I don't think, but it's a line of where we were thinking, yeah.